src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -100,15 +103,53 @@ private void LogCommonPropertiesAndNewLine(IOperation operation)
                 LogString(", IsImplicit");
             }
 
-
             LogString(")");
 
             // Syntax
-            LogString($" (Syntax: {GetSnippetFromSyntax(operation.Syntax)})");
+            LogString($" (Syntax: {GetKind(operation.Syntax)}, {GetSnippetFromSyntax(operation.Syntax)})");
```

Comment:
:bulb: This would be better implemented as a change on its own. If the change highlights a behavior difference in #22726, it should be done in the same way as the previous comment. Otherwise, if the change is just additional information it can be implemented as a PR against the current master branch. It may be helpful to wait until the previous comment is implemented and merged to avoid conflicts with this work.